content/en/developers/metrics/types.md

```diff
@@ -2,298 +2,327 @@
 
-### Customization of tagging
+#### Customization of tagging
 
-This functionality allows you to control tagging for metrics where host-level granularity is not necessary. See the [Distribution Metric page][2] to learn more about whitelist-based tagging control. **Note**: The exclusion of tags with `!` is not accepted with this feature.
+This functionality allows you to control tagging for metrics where host-level granularity is not necessary. See the [Distribution Metric page][1] to learn more about whitelist-based tagging control.
 
-[1]: /developers/metrics/dogstatsd_metrics_submission/#distribution
-[2]: /metrics/distributions
+**Note**: The exclusion of tags with `!` is not accepted with this feature.
+
+
+[1]: https://app.datadoghq.com/metric/distribution_metrics
 {{% /tab %}}
-{{% tab "SET" %}}
+{{< /tabs >}}
+
+### Submission
+
+{{< tabs >}}
+{{% tab "COUNT" %}}
+
+Submit your COUNT type metrics from one of the following sources:
 
-**The `SET` metric type counts the number of unique occurrences of events over a period of time.**
+| Submission Source | Submission Method (python)           | Submission Type | Datadog In-App Type |
+| ----------------- | ------------------------------------ | --------------- | ------------------- |
+| [API][1]          | `api.Metric.send(type="count", ...)` | COUNT           | COUNT               |
+| [DogStatsD][2]    | `dog.count(...)`                     | COUNT           | RATE                |
+| [DogStatsD][2]    | `dog.increment(...)`                 | COUNT           | RATE                |
+| [DogStatsD][2]    | `dog.decrement(...)`                 | COUNT           | RATE                |
+| [Agent check][3]  | `self.count(...)`                    | COUNT           | COUNT               |
+| [Agent check][4]  | `self.monotonic_count(...)`          | COUNT           | COUNT               |
 
-Discover how to submit set metrics:
+**Note**: When submitting a COUNT metric type through DogStatsD, the metric appears as a RATE in-app to ensure relevant comparison across different Agents. Consequently, StatsD counts may appear with a decimal value within Datadog (since they are normalized over a time interval to report units per second).
 
-* [With a custom Agent Check][1]
-* [With DogStatsD][2]
 
-[1]: /developers/metrics/agent_metrics_submission/?tab=set
-[2]: /developers/metrics/dogstatsd_metrics_submission/#set
+[1]: /api/?lang=python#post-timeseries-points
+[2]: /developers/metrics/dogstatsd_metrics_submission/#count
+[3]: /developers/metrics/agent_metrics_submission/?tab=count#count
+[4]: /developers/metrics/agent_metrics_submission/?tab=count#monotonic-count
 {{% /tab %}}
-{{% tab "TIMER" %}}
+{{% tab "RATE" %}}
+
+Submit your RATE type metrics from one of the following sources:
+
+| Submission Source | Submission Method (python)          | Submission Type | Datadog In-App Type |
+| ----------------- | ----------------------------------- | --------------- | ------------------- |
```

Comment:
Alphabetize this table:

| [Agent check][2]  | `self.rate(...)`                    | RATE            | GAUGE               |
| [API][1]          | `api.Metric.send(type="rate", ...)` | RATE            | RATE                |